wasm/mandelbrot.c

```diff
@@ -26,21 +25,23 @@ int iterateEquation(double x0, double y0, int maxiterations) {
   return iterations;
 }
 
-double scale(double domainStart, double domainLength, int screenLength, int step) {
-  return domainStart + domainLength * ((double)(step - screenLength) / (double)screenLength);
+inline double scale(double domainStart, double domainLength, double invScreenLength, int step) {
```

Comment:
changing this function to inline causes the generated wasm binary to expect this function to be imported:

```
(import "env" "scale" (func $scale (param f64 f64 f64 i32) (result f64)))
```

Was that intended?

Reply:
Nope. Hmm, that's strange

Reply:
Ok, I'll remove this. Anyway GCC/Clang should perfectly inline this.